packages/loader/container-loader/src/container.ts

```diff
@@ -1539,7 +1589,9 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
 
         deltaManager.on("disconnect", (reason: string) => {
             this.collabWindowTracker?.stopSequenceNumberUpdate();
-            this.connectionStateHandler.receivedDisconnectEvent(reason);
+            if (!this.closed) {
```

Comment:
Why?

Reply:
Before, the DeltaManager would remove listeners after closing. This listener now catches and could end up sending the `"setConnectionStateSame"` error if we don't check `this.closed`